src/samplers/ebpf/tcp/mod.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+mod statistics;
+
+use self::statistics::Statistic;
+use super::map_from_table;
+use crate::common::{MICROSECOND, PERCENTILES, SECOND};
+use crate::config::*;
+use crate::samplers::{Common, Sampler};
+
+use bcc;
+use bcc::core::BPF;
+use failure::*;
+use logger::*;
+use metrics::*;
+use time;
+
+pub struct Tcp<'a> {
+    bpf: BPF,
+    common: Common<'a>,
+}
+
+impl<'a> Sampler<'a> for Tcp<'a> {
+    fn new(
+        config: &'a Config,
+        metrics: &'a Metrics<AtomicU32>,
+    ) -> Result<Option<Box<Self>>, Error> {
+        debug!("initializing");
```

Comment:
Will the logger print that this is from the ebpf tcp sampler? If not, this should be more specific.

Reply:
Yes, the logger prints the path when set to debug or trace: https://github.com/twitter/rpc-perf/blob/master/logger/src/lib.rs#L71